### ISSUE TITLE
volume doesnt lower with slider for youtube videos

### ISSUE THREAD

**Issue by SkarletShadows** [rocksdanister/lively#1256]
**Describe the bug**

**To Reproduce**
1. Set wall paper to a video
2. try to lower volume
3. wont actually lower output

**Expected behavior**
lower volume makes sound quieter

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/49791649/172500729-23361970-d859-41ea-af4d-c3fbc1f07b14.png)
![image](https://user-images.githubusercontent.com/49791649/172500734-41c4dab5-8671-42df-a44a-f7413b5f1c32.png)

**Desktop (please complete the following information):**
 - OS: windows 11
- youtube video link https://www.youtube.com/watch?v=dEQNn1HBE7U

**Additional context**

**Log file (Important)**
Share the diagnostic files by:
[20220606_164646.txt](https://github.com/rocksdanister/lively/files/8857136/20220606_164646.txt)

**Comment by rocksdanister** [comment]
Yeah, for the browser currently there is only two states for audio - ON/OFF.
If you lower the volume to 0 then it should mute.

**Comment by SkarletShadows** [comment]
Okay, was a bit confusing since it has a slider that goes from 0-100 so
that implies it can be lowered and isnt binary 😅

**Comment by rocksdanister** [comment]
Yes, the slider works 0-100 for video files.
